doc/api/stream.md

```diff
@@ -1508,6 +1508,9 @@ the callback and passing the error as the first argument. This will cause an
 on how the stream is being used.  Using the callback ensures consistent and
 predictable handling of errors.
 
+*Note*: If a Readable stream pipes into a Writable stream when Writable emits an
```

Comment:
Nit: Consider removing `*Note*:` and letting the sentence stand on its own?

Reply:
This seems like something that requires special attention when implementing error handling in Writable streams and `STYLE_GUIDE.md` recommends to mark such things with a `*Note*:`.

Reply:
I've removed the `*Note*`.